package.json

```diff
@@ -34,9 +34,9 @@
     "postis": "^2.2.0"
   },
   "devDependencies": {
-    "electron": "1.4.13",
+    "electron": "*",
```

Comment:
Installing the latest version of electron doesn't really solve the video.captureStream() problem